_includes/configuration/mediaembed_inline_styles.md

```diff
@@ -0,0 +1,6 @@
+### `mediaembed_inline_styles`
+
+This optional setting will inline all styles, instead of using CSS classes, when rendering the embedded snippet. This is useful when the additional CSS classes can't be added to your site. Defaults to `false`.
```

Comment:
This uses `setting` if you're worried.

Reply:
```suggestion
This option will inline all styles, instead of using CSS classes, when rendering the embedded snippet. This is useful when the additional CSS classes can't be added to your site. Defaults to `false`.
```

Reply:
i'm not worried but I'll fix it anyway :shrug: